thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -250,14 +258,14 @@ body {
     display: inline-block;
     color: app-shade(black, 7);
     margin: 0;
-    width: 289px;
 
     .daterangepicker-input {
       background: none;
       border: none;
       box-shadow: none;
       padding: 3px 5px;
       height: auto;
+      width: auto;
```

Comment:
Blocking comment: This seems to be a global change as well. Please make sure that this doesn't break any of the current implementation or create a BEM modifier for `daterangepicker-input`

Reply:
I'll verify